src/main/java/seedu/saveit/model/issue/solution/SolutionLink.java

```diff
@@ -54,7 +54,7 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
             || (other instanceof SolutionLink // instanceof handles nulls
-            && value.equals(((SolutionLink) other).getValue())); // state check
+            && value.equals(((SolutionLink) other).value)); // state check
```

Comment:
maybe it is better to set value as a private field and add a getter method

Reply:
> maybe it is better to set value as a private field and add a getter method

This one `value` is private. I can use `value` instead of `getValue()` because this is in the class, so I can retrieve private attribute here.

Reply:
oh I see